### ISSUE TITLE
Final Review

### ISSUE THREAD

**Issue by shelleyli09** [pangolion/Mental-Project#13]
The project is about mental health. They state that there are many people concealing  their abnormal conditions and refuse to seek health which bring negative influences. The purpose is to predict whether the people in technology having mental problems will seek help or not. They used a dataset with 1524 observations and 78 attributes but did not say where the dataset is from. 

I like that you observed that your original dataset has only 1524 observations but after feature engineering you have 6912 columns and directly using the matrix would have caused overfitting. You also tried a lot of classification models to do the prediction and explained each method in detail which is good. There are also graphs and tables in your report showing your findings. I also like that you added in-text citation that jumps right to the reference on the last reference. 

 
Improvements: 
 

1. In your report, you didn’t explicitly say where your dataset comes from. I would prefer to have a link to it or something equivalent. I know you probably put it in your proposal or midterm report but I think the final report is your final submission to your manager and should include this information expecting your manager to find it in your previous submissions by him/herself.
2. I said before that I like you trying a lot of models to do prediction, however, I think only soft SVM can barely be categorized as one of the methods we learned from class. Even the reason SVM was mentioned is because we were talking about hinge loss which is basically SVM. I don’t think we ever talked about logistic classifier, KNN, Decision Tree, Random forest, and XGBoost in this class. I feel like you are just using methods you learned from other classes to do this project. This wouldn’t be a problem if you are working on a project for a company and can use whatever method you want. However, since this is the final project for ORIE 4741, I feel like you should at least try some methods we learned in class since the purpose of this project is to practice using them on real datasets. 
3. Your dataset consists of 78 attributes — what are these features? Are there age, gender, workplace, whether they are married, how many kids they have, etc.? I would prefer to know what features you are actually dealing with that gives you the prediction. 78 are a lot so there is no need to mention all of them. However, you can at least mention a few that you think are important, or maybe some that you found out are quite important in your later model analysis. For example, since the max_depth for Decision Tree is 13, I doubt that all features are used, maybe you can state what features are chosen by this model? Also, if you don't show me the features, how would you convince me that they are exhaustive?
4. What’s your question? In your introduction you said, “we think it is important to find out the factor that stops people from asking for help from the psychotherapist. This project aims to find out what kind of environment stops people from seeking mental health treatment.” Then you said “In this project, we try to build a model which will predict whether a person will seek mental treatment in the given environment or not.” So you are not finding factors that are  that stops people from asking for help? This also follows from my previous comment — I don’t know what features you have and I don’t know what factors matter. If this is not the question you are trying to answer then don’t pose it. As a reader I would expect to learn what factors affect whether a person asks for help or not after reading the introduction but I get no information after reading your report.
